DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -211,6 +215,56 @@ public void GetRelatedProgressIdForGroupDelegate_returns_null_with_no_record()
             result.Should().BeNull();
         }
 
+        [Test]
+        public void UpdateGroupDescription_updates_record()
+        {
+            using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+            try
+            {
+                // Given
+                const int centerId = 101;
+                const int groupId = 5;
+                const string expectedDescription = "Test group description1";
+
+                // When
+                var isUpdated = groupsDataService.TryUpdateGroupDescription(groupId, centerId, expectedDescription);
+
+                // Then
+                isUpdated.Should().BeTrue();
+                var result = groupsDataService.GetGroupAtCentreById(groupId, centerId);
+                result?.GroupDescription.Should().Be(expectedDescription);
+            }
+            finally
+            {
+                transaction.Dispose();
+            }
+        }
+
+        [Test]
+        public void TryUpdateGroupDescription_with_incorrect_centreId_does_not_update_record()
+        {
+            using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+            try
+            {
+                // Given
+                const int incorrectCentreId = 107;
+                const int groupId = 5;
+                const string expectedDescription = "Test group description";
+
+                // When
+                var isUpdated = groupsDataService.TryUpdateGroupDescription(groupId, incorrectCentreId, expectedDescription);
+
+                //Then
+                isUpdated.Should().Be(false);
+                var result = this.GetGroupDescriptionById(groupId);
+                result?.Should().NotBe(expectedDescription);

```

Comment:
Can we make this line 
`result.Should().BeNull();`
so then we are explicitly testing that the value is the same as how it was.